scripts/flow_track_correl.py

```diff
@@ -0,0 +1,414 @@
+import numpy as np
+from astropy.io import fits
+import glob
+import matplotlib
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import os
+
+
+def read_fits(filepath: str, hdu: int = 1) -> np.ndarray:
+    """
+    Read FITS image array.
+
+    Parameters
+    ----------
+    filepath : str
+        Path to the FITS file.
+    hdu : int, optional
+        Index of the header data unit storing the data array, by default 1.
+
+    Returns
+    -------
+    np.ndarray
+        2D image array from the specified FITS file.
+    """
+    with fits.open(filepath) as hdul:
+        hdul.verify('fix')
+        data = hdul[hdu].data
```

Comment:
Note that our data sometimes come in cubes at levels 2 and 3. So you'll have an array with dimensions (polarization, spatial, spatial). The data I shared was (3, 4096 ,4096). Level 3 polarized files are actually (2, 4096, 4096) because they're in B/pB instead of MZP polarization. Long story short, I added a `layer` parameter to the `read_fits` so that I could get the specific image out with `data = hdul[hdu].data[layer]`.

Reply:
Noted. That's the kind of information I was hoping to get updated on.